src/app/services/crypto.service.ts

```diff
@@ -70,12 +70,12 @@ export class CryptoService implements OnDestroy {
       this.keycloakService = this.injector.get(KeycloakService);
       this.apiEndpointService = this.injector.get(ApiEndpointService);
       this.cryptoFunc = globalThis.GeCoCryptoLib;
-      const haveRightsForPatientList = !!this.keycloakService.getUserRoles().find((role) => role === 'patient_list');
       await this.apiEndpointService.getCall(
         'params'
       ).pipe(
         catchError((err) => {
-          MessageHelper.alert('error', 'Error while getting crypto params.');
+          alert('Error while getting crypto params');
```

Comment:
@romainbou don't hesitate to change this sentence if you think you can formulate better for the user.